tests/api/test_sending.py

```diff
@@ -848,25 +848,8 @@ def fake_conn_pool(acct_id):
     monkeypatch.setattr("inbox.api.ns_api.writable_connection_pool", fake_conn_pool)
 
 
-@pytest.fixture
-def patch_sentry_to_raise(monkeypatch):
```

Comment:
This fixture was not needed for anything, it's a false dependency of the test below